### ISSUE TITLE
Project 2 Test 8 Style Checking

### ISSUE THREAD

**Issue by weiyifei-evelyn** [nus-cs2030/2021-s1#463]
Hi, after passing the test cases for all 7 levels, my code seems to fail test 8 which I think it's style checking (I'm done with Javadoc as well). However, I don't know how I should solve this error.
By the way, I'm writing my code on VS code. 
<img width="1145" alt="Screenshot 2020-11-21 at 9 05 09 PM" src="https://user-images.githubusercontent.com/71199647/99878027-5673f800-2c3d-11eb-9319-af17d9f20859.png">

Please offer me some help. Thank you!

**Comment by weiyifei-evelyn** [comment]
btw, I set the tab size to be 4 spaces already

**Comment by dri12** [comment]
I'm not quite sure if you use other platform. I used vim and added the .vimrc file given in this link https://github.com/nus-cs2030/2021-s1/blob/master/.vimrc 

After that, I set up the checkstyle in the link below and run the it to check for any issues and then corrected it. When I run the checkstyle and no errors detected, then I submitted it to codecrunch.
https://github.com/nus-cs2030/2021-s1/wiki/Setting-Up-Checkstyle
